geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/PropertyLiteral.scala

```diff
@@ -6,7 +6,7 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  */
 
-package org.locationtech.geomesa.accumulo.filter
+package org.locationtech.geomesa.filter
```

Comment:
I moved it back where it was before zcurve because geomesa-kafka-datastore needs it.  There's nothing accumulo specific about it.